en/projekte/2014-07-14-Muenchen-Transparent.html

```diff
@@ -0,0 +1,39 @@
+---
+layout: project
+lab: OK Lab München #needed for event and project aggregation
+imgname: muenchen/Muenchen-Transparent.png
+title: München Transparent
+status: Im Produktivbetrieb
+
+links:
+- url: https://www.muenchen-transparent.de/
+  name: Website
+- url: https://twitter.com/MUCTransparent
+  name: Twitter
+- url: https://github.com/codeformunich/Muenchen-Transparent
+  name: Github
+
+collaborators:
+- name: Tobias Hößl
+  links:
+  - url: https://twitter.com/TobiasHoessl
+    url-name: Twitter
+  - url: http://github.com/catoth
+    url-name: GitHub
+  - url: https://www.hoessl.eu/
+    url-name: Homepage
+- name: Konstantin Schütze
+  links:
+  - url: https://twitter.com/konstinx
+    url-name: Twitter
+  - url: https://github.com/konstin
+    url-name: GitHub
+
+---
+München Transparent ("Munich Transparent") is a plattform allowing the inhabitants of Munich fast and simple insight to municipal politics. This is achieved by presenting the normally difficult to access data from the Munich municipal information system ("Ratsinformationssystem") in a clean and pleasent interface. The data contains over 180,000 documents, ranging from city council proposals over resolution drafts to the daily newspaper of the city council, call "Rathausumschau". That's how you can get to know what the city council is doing at the moment and which are the hot topics in the city council and in the city district councils.
+
+München Transparent does not only want to deliver data, but also explain the procedures in municipal politics. Therefore, there's a coprehensive glossary besides video explaining the way an initative takes bevor it becomes actual policy.
```

Comment:
~~ist Open Source (The Source Code is available on GitHub)~~
is open source (the source code is available on GitHub)